web_frontend/osmaxx/excerptexport/templates/excerptexport/partials/content_file_type_description.html

```diff
@@ -9,6 +9,8 @@
             {% trans 'ESRI Shapefile' %}
         {% elif type == 'fgdb' %}
             {% trans 'ESRI File Geodatabase' %}
+        {% elif type == 'garmin' %}
+            {% trans 'Garmin navigation & map data' %}
```

Comment:
(almost) all builtin templatetags do _autoescaping_ if not otherwise specified, and in this case as well.